index.js

```diff
@@ -160,9 +160,21 @@ gulpSass.logError = function logError(error) {
   this.emit('end');
 };
 
-//////////////////////////////
-// Store compiler in a prop
-//////////////////////////////
-gulpSass.compiler = require('node-sass');
-
-module.exports = gulpSass;
+module.exports = (compiler) => {
+  if (!compiler || !compiler.render) {
+    const message = new PluginError(
+      PLUGIN_NAME,
+      '\n' +
+      'gulp-sass 5 does not have a default Sass compiler; please set one yourself.\n' +
+      'Both the `sass` and `node-sass` packages are permitted.\n' +
+
+        'For example, in your gulpfile:\n\n' +
+        '  var sass = require(\'gulp-sass\')(require(\'sass\'));\n',
+      { showProperties: false },
+    ).toString();
+    process.stderr.write(`${message}\n`);
+    process.exit(1);
+  }
+  gulpSass.compiler = compiler;
+  return gulpSass;
+};
```

Comment:
🔍 Please let me know if I should implement this API some other way!

Reply:
This is outdated because a8de2a0 resulted in some formatting changes, but the implementation is the same